docs/api/routing.rst

```diff
@@ -394,6 +394,61 @@ A custom router is any class that implements the following interface:
 
             """
 
+Suffixed Responders
+-------------------
+
+While Falcon encourages the REST architectural style, it is flexible enough to accomodate other 
+paradigms. Consider the task of building an API for a calculator which can both add and subtract 
+two numbers. You could implement the 
+following:
+
+.. code:: python
+
+    class Add():
+        def on_get(self, req, resp):
+            resp.body = str(int(req.get_param('x')) + int(req.get_param('y')))
```

Comment:
Ah, just a late nitpick... We could simplify these casts to `int` by using Falcon's functionality out of the box: see [Request.get_param_as_int()](https://falcon.readthedocs.io/en/stable/api/request_and_response.html#falcon.Request.get_param_as_int) :slightly_smiling_face:

Reply:
Updated